README.md

```diff
@@ -61,11 +61,15 @@ Project spec & rubric can be found [here](https://frontend.turing.io/projects/mo
 [Hang in there group project](https://joanafbrito.github.io/hang-in-there-boilerplate/)
 
 ## Images/gifs:
-![Main page Poster](https://user-images.githubusercontent.com/82066350/119276133-fb13ca00-bbcd-11eb-9071-83c7e934e71c.png)
+![Main page poster](https://user-images.githubusercontent.com/82066350/119276133-fb13ca00-bbcd-11eb-9071-83c7e934e71c.png)
```

Comment:
I noticed that this photos include part of my browser window including the tabs. Can you crop these to remove that?

Reply:
William, that was a good  catch on the images, I have re-cropped the screenshots.